dashboard/app/models/sections/section.rb

```diff
@@ -687,18 +687,37 @@ def assigned_ai_chat?
       pswai-pilot-2024
     ]
 
-    gen_ai_course = 'exploring-gen-ai-2024'
+    gen_ai_courses = %w[
+      exploring-gen-ai-2024
+      exploring-gen-ai-2025
+      foundations-gen-ai-2025
+      customizing-llms-2025
+    ]
 
     csaif_courses = %w[
       computer-systems-and-devices-2024
       programming-fundamentals-2024
       programming-fundamentals-aitutor-2024
       networks-and-the-internet-2024
       problem-solving-with-ai-2024
+      artificial-intelligence-foundations-2025
+      computing-foundations-for-a-digital-age-2025
+      foundations-of-ai-programming-2025
+      ai-and-the-systems-that-power-it-2025
+      the-fabric-of-the-internet-and-ai-2025
+      cybersecurity-and-global-impacts-2025
+      insights-from-data-and-ai-2025
+    ]
+
+    # In order to support an organizational event.
+    other_courses = %w[
+      codechella2025
     ]
```

Comment:
I think we've discussed in the past about moving this system to DCDO, at least in the interim while we design more a holistic solution. Would it be worth doing that to support one-off courses like this (as a separate task)?